src/applications/claims-status/components/WhatWereDoing.jsx

```diff
@@ -0,0 +1,59 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import { Link } from 'react-router';
+
+const statusMap = {
+  CLAIM_RECEIVED: 'Step 1 of 5: Claim received',
+  INITIAL_REVIEW: 'Step 2 of 5: Initial review',
+  EVIDENCE_GATHERING_REVIEW_DECISION:
+    'Step 3 of 5: Evidence gathering, review, and decision',
+  PREPARATION_FOR_NOTIFICATION: 'Step 4 of 5: Preparation for notification',
+  COMPLETE: 'Step 5 of 5: Closed',
+};
+
+function getStatusDescription(status) {
+  return statusMap[status];
+}
+
+function WhatWereDoing({ claim }) {
+  const { status } = claim.attributes;
+
+  const humanStatus = getStatusDescription(status);
+
+  return (
+    <div className="what-were-doing-container vads-u-margin-bottom--4">
+      <h3 className="vads-u-margin-bottom--3">What we’re doing</h3>
+      <va-card uswds="false">
+        <h4 className="vads-u-margin-top--0 vads-u-margin-bottom--1">
```

Comment:
I would just leave the bottom margin alone on this one

Reply:
@skylerschain if we leave the bottom margin alone on the h4 then it would be 8.5 px instead of 8px are you okay with changing it to that or should we keep it at 8px?

Reply:
Talked to @skylerschain and hes going to talk to the design team as we should be doing multiples of 8px for the spacing so 8.5px by default seems inconsistent. For now leave as 8px.